unison-src/transcripts/contrabilities.md

```diff
@@ -3,6 +3,6 @@
 ```
 
 ```unison
-f : 'a -> Nat
```

Comment:
@pchiusano Why this change?

Reply:
Hmm, I'm honestly not sure the purpose of this transcript and was just mechanically translating, since `'blah` doesn't allow effects and the existing output did allow effects. But looking at it now, its purpose might be to see if arrows in contravariant position get inferred as ability polymorphic? I switched it to use `(() -> a)` here, and it infers as `(() ->{g} a)`.